eftcamb/01_EFT_def.f90

```diff
@@ -57,6 +57,23 @@ module EFTDef
     logical , parameter :: DebugEFTCAMB = .false.       !< EFTCAMB debug flag.This will turn on printing of many things to aid debugging the code.
 #endif
 
```

Comment:
I would remove the following. Stability choices should be a property of EFTCAMB and not defined at compile time.
Look at 11_EFTCAMB_main, there you find the stability selection flags!